navigation/libandroid-navigation/src/main/res/layout/layout_notification_default.xml

```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
-
 <LinearLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Possible to move these attributes to the `LinearLayout` below and then remove?  The child of this parent is another `LinearLayout` and removing the parent would prevent another pass for drawing